packages/db/src/translate/file.ts

```diff
@@ -16,6 +16,19 @@ const getKey = (file: ApiFile) => {
   return undefined;
 };
 
+export const translatePostgresFileToApiFile = (
+  filePgRecord: PostgresFileRecord
+): Omit<ApiFile, 'granuleId'|'createdAt'|'updatedAt'> => ({
```

Comment:
So we're updating the timestamp fields because they're not present in the api *model* granules.files schema?   This

Should we be including them given we're not returning the File at all via any other route now?    If not, should we be using the ApiFile type?

Reply:
So I wasn't treating `ApiFile` as 100% the source of truth for what our API actually returns. In this case I was looking at our API docs to see what a file actually looks like:

https://nasa.github.io/cumulus-api/#retrieve-granule

Between that and the `ApiFile`. That's probably not the right call though. The return should match the `ApiFile`.

https://github.com/nasa/cumulus/pull/2415/commits/6e1aa9d40c84523a4bfb780d22901093be127c96
https://github.com/nasa/cumulus/pull/2415/commits/d69df57f836fd842415b3eed5bd3154a04d5f8e4

I've done that here BUT I am still omitting `granuleId`. I think this will be a breaking change but I'm doing it because we can't use just `granuleId` for any granule lookup. it seemed potentially dangerous to return it. Thoughts?